Softeq.XToolkit.Bindings.iOS/Gestures/GestureRecognizerBehaviour.cs

```diff
@@ -1,14 +1,15 @@
 // Developed by Softeq Development Corporation
 // http://www.softeq.com
 
-using System.Windows.Input;
+
+using Softeq.XToolkit.Common.Commands;
 using UIKit;
 
 namespace Softeq.XToolkit.Bindings.iOS.Gestures
 {
-    public abstract class GestureRecognizerBehavior
+    public abstract class GestureRecognizerBehaviour<T>
```

Comment:
Probably we should restrict T to be derived from UIGestureRecognizer. This would allow us to change the method AddGestureRecognizer to generic:
protected void AddGestureRecognizer(UIView target, T gestureRecognizer)

Reply:
fixed